### ISSUE TITLE
Release new version to support Go 1.19

### ISSUE THREAD

**Issue by lcheylus** [mr-karan/doggo#62]
I'm creating an OpenBSD port for doggo with the latest version (0.5.4). I'm using Go 1.19 to build my port (currently supported version on OpenBSD-current / future version 6.3).

But I have an error during my build with Go version 1.19 because `github.com/lucas-clemente/quic-go` module v0.26.0 (version required for `doggo` v0.5.4) is not supported with Go 1.19 : 
```
$ go build
(...)
github.com/lucas-clemente/quic-go/internal/qtls
# github.com/lucas-clemente/quic-go/internal/qtls
../../../go/pkg/mod/github.com/lucas-clemente/quic-go@v0.26.0/internal/qtls/go119.go:6:13: cannot use "quic-go doesn't build on Go 1.19 yet." (untyped string constant) as int value in variable declaration
```

Please could you release a new version (0.5.5 ?) of doggo for the latest requirements supported with Go 1.19 ?

**Comment by mr-karan** [comment]
Thanks for creating an OpenBSD port, really appreciated.

Released 0.5.5: https://github.com/mr-karan/doggo/releases/tag/v0.5.5

**Comment by lcheylus** [comment]
Thanks, my OpenBSD for doggo v0.5.5 builds correctly with Go v1.19.
